text/0000-compressed-nft-standard.md

```diff
@@ -0,0 +1,195 @@
+1. Mint the item being claimed.
+
+## Augmenting API version 1
+
+Once a user has obtained and decoded the URI using the `get_api_info` get method, verified the version (this standard covers version 1 only) and added the `/v1` postfix, they can use the following methods:
+
+(`:arg` means an argument with name `arg`)
+
+##### 1. `/item/:index`
+
+Returns full information about a particular item including the proof Cell required to claim it.
+
+MUST return a JSON object with the following fields:
+
+| Name | Type | Description |
+| --- | --- | ---|
+| item | object | The item data serialized as a JSON object as discussed in the Item data section |
+| proof_cell | string | The proof Cell that must be used in the `claim` request to claim this item. Serialized as base64 BoC |
+
+Other fields MAY be returned.
+
+Example:
+```json
+{
+	"item": {
+		"metadata": {
+			"owner": "UQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAJKZ",
+			"individual_content": "te6cckEBAQEACAAADDAuanNvbuTiyMU="
+		},
+		"index": 0
+	},
+	"proof_cell": "te6cckEBBgEAeQACAAEDAUOAFa1KqA2Oswxbo4Rgh/q6NEaPLuK9o3fo1TFGn+MySjqQAgAMMi5qc29uAUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQBQF4S3GMDJY/HoZd6TCREIOnCaYlF23hNzJaSsfMd1S7nBQAA8muEeQ=="
+}
+```
+
+##### 2. `/items?offset=:offset&count=:count`
+
+Returns information about multiple items at once.
+
+API implementations MAY impose a limit on the `count` argument. If `count` exceeds the imposed limit, the API implementations MUST treat that request as if it had `count` set to the imposed limit, instead of the actual `count`.
+
+MUST return a JSON object with the following fields:
+
+| Name | Type | Description |
+| --- | --- | --- |
+| items | object[] | An array of at most `count` Item data objects or nulls (if there are gaps in the item list), the first one having the index `offset` |
+| last_index | number | The index of the last item in the item list. Enumeration of items past this index cannot be done |
+
+Other fields MAY be returned.
+
+Example: (with `offset` 0, `count` 4, with the item list having an item index 0 and an item at an index greater than 3 with gaps in between)
+```json
+{
+	"items": [
+		{
+			"metadata": {
+				"owner": "UQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAJKZ",
+				"individual_content": "te6cckEBAQEACAAADDAuanNvbuTiyMU="
+			},
+			"index": 0
+		},
+		null,
+		null,
+		null
+	],
+	"last_index": 4
+}
+```
+
+##### 3. `/state`
+
+Returns general information about this API instance.
+
+MUST return a JSON object with the following fields:
+
+| Name | Type | Description |
+| --- | --- | --- |
+| last_index | number | The index of the last item in the item list. Enumeration of items past this index cannot be done |
+| address | string | The address of the collection smart contract that this API augments. Encoded as user-friendly url-safe address string |
+
+Other fields MAY be returned.
+
+Example:
+```json
+{
+	"last_index": 0,
+	"address": "UQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAJKZ"
```

Comment:
same - better to use raw form for avoiding problems. for example nft collection it's 100% a smartcontract but in documentation you used non-bounce form

Reply:
@FIFA_wor1d